.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+These issues are only for bugs related to the software navitia, problems on navitia.io related to the data must be addressed on https://groups.google.com/forum/#!forum/navitia
+
+**To Reproduce**
+An url of navitia.io where the problem is visible. If the case appear on your navitia instance the datasets that you used.
```

Comment:
```suggestion
Please provide a navitia.io URL that reproduces the problem (you cal also refer to our [Playground](http://canaltp.github.io/navitia-playground/play.html)). In the case of a custom Navitia instance, you may provide access the datasets being used.
```

Reply:
@nonifier If we mention the playground, we must find a way for people to pay attention that no token is provided in URL.